src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/ScanRunModel.java

```diff
@@ -0,0 +1,115 @@
+package org.jboss.xavier.analytics.pojo.output.workload.summary;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.ColumnResult;
+import javax.persistence.ConstructorResult;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedNativeQuery;
+import javax.persistence.SqlResultSetMapping;
+
+@SqlResultSetMapping(
+        name = "mappingScanRunModels",
+        classes = @ConstructorResult(
+                targetClass = ScanRunModel.class,
+                columns = {
+                        @ColumnResult(name = "target", type = String.class),
+                        @ColumnResult(name = "date", type = Timestamp.class),
+                        @ColumnResult(name = "type", type = Boolean.class)
+                }
+        )
+)
+
+@NamedNativeQuery(
+        name = "ScanRunModel.calculateScanRunModels",
+        query = "select provider as target, creation_date as date, ssa_enabled as type from workload_inventory_report_model  where analysis_id = :analysisId group by provider order by provider;",
```

Comment:
I've tried the query and it complains about not having the `creation_date` as part of the `group by` statement.
I think the query should change to
- have just the day of the `creation_date` (because each `creation_date` will have a different value of time) in the `group by` statement
- calculate the value of the `type` field: if some VMs have `ssa_enabled` to `false` and other VMs have it to `true`, then we should do an `OR` to say that, in that analysis, SSA have been enabled
```suggestion
        query = "select provider as target, cast(creation_date as date) as date, bool_or(ssa_enabled) as type from workload_inventory_report_model  where analysis_id = :analysisId group by provider, date order by provider;",
```